components/docs-chef-io/content/automate/reports.md

```diff
@@ -65,97 +63,95 @@ Recipe
 
 ### Deep Filtering
 
-Deep Filtering allows you to view the state of your infrastructure from the perspective of a single profile, or a single profile and one of its child controls.  
-Adding a profile filter will modify all compliance reports screens to display the status of that profile across all nodes executing that profile. Adding a filter for a child control of that profile will modify all compliance reports screens to display the status of that child control.
+Deep filtering shifts the perspective of the results from the node to the profile and its associated control.
+Use deep filtering to see the compliance reporting for an entire profile or an individual control within a profile.
 
-Please note the only supported filters for deep filtering are:
+Deep filtering supports filtering for:
 
     * one profile
-    * one profile and one of its child controls
+    * one profile and one of its associated controls
 
 ### Waivers
 
-A node's waived status appears if applicable in displays where a node's status appears in Chef Automate. 
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate.
 The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
-_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively. 
-Select the _Waived_ status filter to display only the respective Node or Profile reporting with that status.
+_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively.
+Selecting the _Waived_ status filter displays the respective Node or Profile reporting with the `waived` status.
 Hover over the control's Waived icon under the Node Status column in _Controls_ to view more details about the waiver applied to the control.
 
-Use Chef InSpec to configure [waivers](https://docs.chef.io/inspec/waivers/).
+Use Chef InSpec to configure [waivers](https://www.inspec.io/docs/reference/waivers/).
```

Comment:
The original link is correct here.
```suggestion
Use Chef InSpec to configure [waivers](https://docs.chef.io/inspec/waivers/).
```

Reply:
Thank you for catching that, Ian. I checked and the rest of the links should be okay.